src/api/utilities.ts

```diff
@@ -0,0 +1,95 @@
+import { connect, Utils } from '..';
+import { Client } from '../client';
+
+let saveClient: (clientData: string | null) => void;
+let loadClient: () => Client | undefined;
+
+export const setup = async ({
```

Comment:
Feels like there should be some comments for these functions

Reply:
[235fd9e](https://github.com/GridPlus/gridplus-sdk/pull/525/commits/235fd9e7f44a6044fb7e5e6f850b64af23c5b4fc)